.github/workflows/veracode-scan.yml

```diff
@@ -0,0 +1,49 @@
+name: Veracode scan
+
+on:
+  push:
+    branches: ['\d+.x']
+  pull_request:
+    branches: ['\d+.x']
```

Comment:
Do we need the veracode scan be triggered on branch pushes and pull requests?

Reply:
I think we can probably get rid of the pull request trigger and keep the push trigger in order to mirror the previous scan pipelines

Reply:
oh i just assumed that it would be the same as the TFS pipeline (i.e. which i think has branch push and schedule trigger), but i guess we can change this to just schedule trigger

Reply:
ok i just created a PR for both JS client libraries 
JS client core -> https://github.com/Laserfiche/lf-api-client-core-js/pull/69
JS repo client ->  https://github.com/Laserfiche/lf-repository-api-client-js/pull/88